src/types/User.ts

```diff
@@ -0,0 +1,23 @@
+export default interface User {
+  id: number,
+  name: string,
+  username: string,
```

Comment:
Santa Maria! 😱 
You dont need to write type for every property. Only those that you'll use in your project. But in general it's outstanding because now you can use autofill for every property of User 😅

Reply:
Yep, I just copied user from array and just values with `string`